gsa/src/web/components/powerfilter/__tests__/tasktrendgroup.js

```diff
@@ -0,0 +1,123 @@
+/* Copyright (C) 2019 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import React from 'react';
+
+import {
+  render,
+  fireEvent,
+  queryAllByTestId,
+  getByTestId,
+} from 'web/utils/testing';
+
+import TaskTrendGroup from 'web/components/powerfilter/tasktrendgroup';
+import Filter from 'gmp/models/filter';
+
+const openSelectElement = element => {
+  const openButton = getByTestId(element, 'select-open-button');
+  fireEvent.click(openButton);
+};
+
+const getItemElements = baseElement => {
```

Comment:
Could you import openSelectElement and getItemElements from web/components/form/__tests__/select instead?